eng/pipelines/public.yml

```diff
@@ -8,38 +8,45 @@ pr:
 
 trigger: none
 
+resources:
+  repositories:
+  - repository: main
+    type: github
+    name: dotnet/workload-versions
+    # TODO: Update when the changes are merged to the main branch.
```

Comment:
I put the TODOs here, but updating the branch names here will be done when everything is reviewed and ready to merge. Should be the last commit on this branch, prior to closing out this PR.